doc/linux/SELINUX.md

```diff
@@ -0,0 +1,46 @@
+# Dinit SELinux Awareness
+
+Dinit has support for basic SELinux awareness. This document is intended to
+outline the extent and inner workings of Dinit's SELinux awareness. The reader
+is assumed to be knowledgeable about the basics of [SELinux](https://github.com/SELinuxProject/selinux-notebook) and Dinit.
+
+Dinit needs to be built with SELinux support (see [BUILD](/BUILD)) to enable the features that are
+mentioned in this document.
+
+## Loading the system SELinux policy
+When booted as the system init system, dinit by default will attempt to load the
+system's SELinux policy and transition itself to a context specified by that policy
+if not already done so in earlier boot (e.g. by an initramfs). This behaviour may be
+disabled by passing dinit the `--disable-selinux-policy` flag. As dinit will always
+be PID1 in this senario, this can be done by appending the flag to the kernel cmdline.
+
+If not already mounted in earlier boot (e.g. by an initramfs), dinit will mount `/sys`,
+and selinuxfs (typically `/sys/fs/selinux`) during the call to `selinux_init_load_policy(3)`.
```

Comment:
This is too specific to implementation details. How should the user know when/if `selinux_init_load_policy` is called? What is this telling them? What do they need to know instead? (eg. that it happens before any services are started).

Reply:
Should be sorted, let me know if the updated documentation is okay

Reply:
See the comments I made since.